### PR TITLE
musig-spec: Improve writing in Motivation, Design

### PR DESCRIPTION
**Pull request by real-or-random**

### CODE REVIEW

File: doc/musig-spec.mediawiki [hunk 1]
```diff
@@ -24,34 +24,38 @@ This document is licensed under the 3-clause BSD license.
...
+The primary motivation for MuSig2 is the activation of Taproot ([https://github.com/bitcoin/bips/blob/master/bip-0341.mediawiki BIP341]) on the Bitcoin network, which introduced the ability to authorize transactions with Schnorr signatures.
 This standard allows the creation of aggregate public keys that can be used in Taproot outputs.
-Such outputs are indistinguishable for a blockchain observer from regular, single-signer outputs but are actually controlled by multiple signers.
-Moreover, by tweaking an aggregate key, the shared Taproot output can have script spending paths that are hidden unless used.
+
+The on-chain footprint of a MuSig2 Taproot output is a single BIP340 public key, and a transaction spending the output only requires a single signature cooperatively procuded by all signers. This is '''more compact''' and has '''lower verification cost''' than each signer providing an individual public key and signature, as would be required by an ''n-of-n'' policy implemented using <code>OP_CHECKSIGADD</code> as introduced in ([https://github.com/bitcoin/bips/blob/master/bip-0342.mediawiki BIP342]).
```
Comment (flack) [inline doc/musig-spec.mediawiki:34]: procuded => produced

Comment (real-or-random) [reply doc/musig-spec.mediawiki]: fixed

File: doc/musig-spec.mediawiki [hunk 2]
```diff
@@ -24,34 +24,38 @@ This document is licensed under the 3-clause BSD license.
 
 === Motivation ===
 
-MuSig2 is a multi-signature scheme that allows multiple signers to create a single aggregate public key and cooperatively create a single Schnorr signature for the aggregate key and a message.
-This is more space-efficient and has lower verification costs than each signer providing an individual public key and signature.
-Since MuSig2 is not a threshold-signature scheme, the cooperation of ''all'' signers involved in key aggregation is required to produce a signature.
+MuSig2 is a multi-signature scheme that allows multiple signers to create a single aggregate public key and cooperatively create ordinary Schnorr signatures valid under the aggregate key. 
+Signing requires interaction between ''all'' signers involved in key aggregation.
+(MuSig is a ''n-of-n'' multi-signature scheme and not a ''t-of-n' threshold-signature scheme.)
```
[the remark this answers is not in the document]
Comment (real-or-random) [reply doc/musig-spec.mediawiki]: fixed

**Review by jonasnick — COMMENTED** [review]
ACK d903c09fd2087684281239187346cc8856b4fdca

**Pull request merged**